docs/templates/tutorials/distributed-tuning.md

```diff
@@ -0,0 +1,122 @@
+# Distributed Tuning
+
+Keras Tuner makes it easy to perform distributed hyperparameter search. No changes to your code are needed to scale up from running single-threaded locally to running on dozens or hundreds of workers in parallel. Distributed Keras Tuner uses a chief-worker model. The chief runs a service to which the workers report results and query for the hyperparameters to try next. The chief should be run on a single-CPU worker (or alternatively as a separate process on one of the workers).
+
+### Configuring distributed mode
+
+Configuring distributed mode for Keras Tuner only requires setting three environment variables:
+
+**KERASTUNER_TUNER_ID**: This should be set to "chief" for the chief process. Other workers should be passed a unique ID (by convention, "tuner0", "tuner1", etc).
+
+**KERASTUNER_ORACLE_IP**: The IP address of the chief service.
+
+**KERASTUNER_ORACLE_PORT**: The port that the chief service should run on.
```

Comment:
Please mention that this is something the user is free to choose themselves (otherwise people will ask, "where do I find this value?"), but that they should choose a port that is accessible (most people will be running on EC2 or GCP, where accessible ports have to be configured beforehand). Also explicitly mention the protocol that should be allowed on this port.

Reply:
Done